ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/NodeCve/AffectedNodesTable.tsx

```diff
@@ -131,75 +126,72 @@ function AffectedNodesTable({ tableState }: AffectedNodesTableProps) {
-                    const isFixable = getAnyVulnerabilityIsFixable(vulns);
-                    const { cvss, scoreVersion } = getHighestCvssScore(vulns);
-
-                    return (
-                        <Tbody key={id} isExpanded={isExpanded}>
-                            <Tr>
-                                <Td
-                                    expand={{
-                                        rowIndex,
-                                        isExpanded,
-                                        onToggle: () => expandedRowSet.toggle(id),
-                                    }}
-                                />
-
-                                <Td dataLabel="Node">
-                                    <Link to={getNodeEntityPagePath('Node', id)}>
-                                        <Truncate position="middle" content={name} />
-                                    </Link>
-                                </Td>
-                                <Td dataLabel="CVE severity" modifier="nowrap">
-                                    <VulnerabilitySeverityIconText severity={topSeverity} />
-                                </Td>
-                                <Td dataLabel="CVE status" modifier="nowrap">
-                                    <VulnerabilityFixableIconText isFixable={isFixable} />
-                                </Td>
-                                <Td dataLabel="CVSS score" modifier="nowrap">
-                                    <CvssFormatted cvss={cvss} scoreVersion={scoreVersion} />
-                                </Td>
-                                <Td dataLabel="Cluster">
-                                    <Truncate position="middle" content={node.cluster.name} />
-                                </Td>
-                                <Td dataLabel="Operating system">{node.operatingSystem}</Td>
-                                <Td dataLabel="Affected components">
-                                    {nodeComponents.length === 1
-                                        ? nodeComponents[0].name
-                                        : pluralize(nodeComponents.length, 'component')}
-                                </Td>
-                            </Tr>
-                            <Tr isExpanded={isExpanded}>
-                                <Td />
-                                <Td colSpan={colSpan - 1}>
-                                    <ExpandableRowContent>
-                                        <NodeComponentsTable />
-                                    </ExpandableRowContent>
-                                </Td>
-                            </Tr>
-                        </Tbody>
-                    );
-                })}
+            <TbodyUnified
+                tableState={tableState}
+                colSpan={colSpan}
+                emptyProps={{
+                    message: 'No nodes are detected to have been reported for this CVE',
```

Comment:
Off the subject of this refactoring, my mind cannot parse in this sentence for relationship of node and CVE.

Reply:
Agreed, I'll change it to `'There are no nodes that are affected by this CVE'` which I believe is easier to parse.